src/main/java/org/xstefank/check/TemplateChecker.java

```diff
@@ -91,4 +92,38 @@ private static FormatConfig readConfig() {
         }
     }
 
+    public boolean shouldSkip(JsonNode payload) {
```

Comment:
better, but still I think that this code is not particularly tied with TemplateChecker and should be moved to separate class